src/DDTrace/Sampling/ConfigurableSampler.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace DDTrace\Sampling;
+
+use DDTrace\Configuration;
+use DDTrace\Contracts\Span;
+
+/**
+ * A sampler configurable using global a global configuration parameter.
+ */
+class ConfigurableSampler implements Sampler
```

Comment:
Just to lock things down a bit. :)
```suggestion
final class ConfigurableSampler implements Sampler
```